bom/deployment/pom.xml

```diff
@@ -43,6 +43,14 @@
                 <scope>import</scope>
             </dependency>
 
+            <!-- Qpid JMS -->
+            <dependency>
+                <groupId>org.amqphub.quarkus</groupId>
+                <artifactId>quarkus-qpid-jms-parent</artifactId>
+                <version>${quarkus-qpid-jms.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
```

Comment:
1) Given that this BOM imports the runtime one which in its turn imports `quarkus-qpid-jms-parent`, this import is not effective.
2) In Quarkus we differentiate between "runtime" and "deployment" dependencies. What does `quarkus-qpid-jms-parent` contain?

Reply:
The quarkus-qpid-jms-parent currently has the qpid-jms-client, the quarkus-qpid-jms and quarkus-qpid-jms-deployment modules for the extension, and quarkus-build-parent: https://github.com/amqphub/quarkus-qpid-jms/blob/0.11.2/pom.xml#L91-L120

I mainly copied that from the camel example above, but I admit I didn't pay close attention to what that contains.

Reply:
If you do have a runtime and deployment BOM then please import your runtime BOM into the runtime platform BOM and the deployment one into the deployment platform BOM.

Reply:
The extension doesn't have boms, the referenced https://github.com/amqphub/quarkus-qpid-jms/blob/0.11.2/pom.xml#L91-L120 is where the versions are managed currently.

The dependencies except the client jar and extension jars are already (or will be in 1.3.0) managed by the core Quarkus bits, if that matters.

Reply:
Ok, `quarkus-build-parent` should not be imported in neither of the platform BOM.
Just import
```
    <dependency>
        <groupId>org.amqphub.quarkus</groupId>
        <artifactId>quarkus-qpid-jms</artifactId>
        <version>YOUR_VERSION</version>
      </dependency>
```
in the runtime platform BOM. And
```
      <dependency>
        <groupId>org.amqphub.quarkus</groupId>
        <artifactId>quarkus-qpid-jms-deployment</artifactId>
        <version>YOUR_VERSION</version>
      </dependency>
```
in the deployment platform BOM.
And I guess you'll want the client in the runtime, right?

Reply:
Ok I can change that. To add the client to the platform runtime bom explicitly I would need a further version property, with both needing changed for updates, is that ok?

Reply:
Either that or create your runtime BOM and import it into the runtime platform BOM (perhaps that would look simpler and may turn out to be a good practice for every contributor). The maintenance of that will be your (team's) responsibility.

Reply:
I've pushed an update reflecting the original suggestions since that works with whats available now. I'll be the one raising updates anyway, and I'd still need to figure out what would go into a bom and create+release it, which can still come later.